EAGLE Libraries/HyTechExternal.lbr

```diff
@@ -270,39 +262,40 @@
 <devices>
 <device name="">
 <technologies>
-<technology name="MICROFIT">
+<technology name="MICROFIT_HORIZONTAL">
 <attribute name="DKPN" value="WM1845-ND"/>
 <attribute name="MANUFACTURER" value="Molex"/>
-<attribute name="MOPN" value="538-43645-0200"/>
-<attribute name="MPN" value="0436450200"/>
+<attribute name="MOPN" value="Molex"/>
+<attribute name="MPN" value="436450200"/>
 </technology>
 </technologies>
 </device>
 </devices>
 </deviceset>
-<deviceset name="FUSE_*" prefix="O">
+<deviceset name="FUSE_*">
 <description>Fuse
 &lt;br&gt;
 &lt;ul&gt;
 &lt;li&gt;&lt;a href="https://www.mouser.com/datasheet/2/240/Littelfuse_MINI_Datasheet-1077557.pdf"&gt;MINI Blade&lt;/a&gt;&lt;/li&gt;
 &lt;/ul&gt;</description>
 <gates>
-<gate name="G$1" symbol="ORDERING" x="0" y="0"/>
 </gates>
 <devices>
 <device name="">
 <technologies>
-<technology name="MINI_BLADE_2">
```

Comment:
not really sure what happened to this file but a lot of these changes shouldn't be here